ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -4656,6 +4668,19 @@ HLSAFS:
   implements:
   - CONTROL
 
+HLDCAFS:
+  description: " VAV type with high and low limit Cooling setpoints"
```

Comment:
Please make this more general (remove VAV) and make the "C" in cooling lowercase.

Reply:
Found a proper canonical type closing the request